suite-native/state/src/extraDependencies.ts

```diff
@@ -12,12 +12,13 @@ import { NativeUsbTransport } from '@trezor/transport-native';
 const protobufMessages = require('@trezor/protobuf/messages.json');
 
 const transports = Platform.select({
-    ios: ['UdpTransport'],
+    ios: ['BridgeTransport', 'UdpTransport'],
```

Comment:
bridge works on ios?

Reply:
Yes it does. `trezor-user-env` (emulator + bridge) works with the iOS simulator.

Reply:
I reworked this little bit more, not there are different transport for emulators and real devices